cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -127,7 +131,9 @@ data GenesisCreateTestNetDataCmdArgs era = GenesisCreateTestNetDataCmdArgs
   , outputDir :: !FilePath
   -- ^ Directory where to write credentials and files.
   }
-  deriving Show
+
+instance Show GenesisCreateTestNetDataCmdArgs where
+  show _ = "GenesisCreateTestNetDataCmdArgs"
```

Comment:
@palas> it doesn't derive anymore because there's an existential type in `GenesisCreateTestnetDataCmdArgs` now: 

![image](https://github.com/user-attachments/assets/ec5e51fa-d75f-41f3-80a6-fc41ccd93f8f)